### PR TITLE
Set deploy branch to 1.1/release and add file to gitignore

### PR DESCRIPTION
**Pull request by ob6160**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ npm-debug.log
 yarn-error.log
 screenshots
 .idea
+package-lock.json
```
Comment (katjam) [inline .gitignore]: Why have you added and ignored package-lock? If there is one, we'd want to include it.

**Review by katjam — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ob6160) [reply .gitignore]: I didn't add it - npm added it itself at some point. I'll remove the ignore and we can make a ticket about including it.

Comment (katjam) [reply .gitignore]: Did you run an update?

Comment (ob6160) [reply .gitignore]: Possibly. I can't remember though.

**Review by katjam — APPROVED** [review]
:goat: But DON'T merge until after filming.

**Pull request merged**